cluster/examples/kubernetes/ceph/object-openshift.yaml

```diff
@@ -95,3 +95,14 @@ spec:
     # Configure the pod liveness probe for the rgw daemon
     livenessProbe:
       disabled: false
+
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: rook-ceph-rgw-my-store
+  namespace: rook-ceph  # namespace:cluster
+spec:
+  to:
+    kind: Service
+    name: rook-ceph-rgw-my-store
```

Comment:
Good to comment-doc this with something like
```yaml
# The name of the RGW service is in the form 'rook-ceph-rgw-<objectstore-name>'
```

Reply:
Done.